test/tests/functional/pbs_conf_var.py

```diff
@@ -62,20 +62,18 @@ def test_sched_host_name_envvar(self):
             self.server.hostname,
             confs={'PBS_SCHEDULER_HOST_NAME': conf['PBS_SERVER']})
         now = int(time.time())
-        self.server.start()
+        self.server.restart()
```

Comment:
Please remove the call to self.server.stop()/isUp() above, this call to restart() will handle it.  The part of your code I disliked was the call to isUp() after the stop.  It's going to add 20s to the test that we don't really need.

Reply:
If the pid is gone isUp() will return false on the first iteration of the loop, so it won't add 20s.
But I do agree not stop/starting and instead just restarting is better.